pwa/src/components/logs/logModal/LogModal.tsx

```diff
@@ -34,26 +34,39 @@ const LogModal: React.FC<LogModalProps> = ({ log }) => {
                     <tr>
                       <th>Status</th>
                       <td>
-                        <LabelWithBackground
-                          label={log?.responseStatusCode}
-                          type={statusClass}
-                        />
+                        <LabelWithBackground label={log?.responseStatusCode} type={statusClass} />
                       </td>
                     </tr>
                     <tr>
                       <th>Type</th>
-                      <td>
-                        {log?.type === "in" ? "Incoming" : "Outcoming"}
-                      </td>
+                      <td>{log?.type === "in" ? "Incoming" : "Outcoming"}</td>
                     </tr>
                     <tr>
                       <th>Call ID</th>
-                      <td>{log?.callId}</td>
+                      <td>
+                        <Link
+                          to={`/calls/${log?.callId}`}
+                          aria-label="Close"
+                          type="button"
+                          data-bs-dismiss="modal"
+                          onClick={() => navigate(`/calls/${log?.callId}`)}
+                        >
+                          {log?.callId}
+                        </Link>
+                      </td>
```

Comment:
This odd way of linking is required for the `Modal` to close on redirect. However, we should refactor this at some point (this is being used in more places, so therefore I did not refactor it in this pull request).